drivers/usb_c/tcpc/ucpd_stm32.c

```diff
@@ -1022,43 +1081,42 @@ static void ucpd_isr(const struct device *dev_inst[])
 	 * Multiple UCPD ports are available
 	 */
 
-	uint32_t ucpd_base;
+	uint32_t sr1;
+	uint32_t sr2;
```

Comment:
Would use `sr0/1` to avoid confusion between device instances and IP names

Reply:
done